src/test/groovy/com/github/rodionovsasha/commentservice/controllers/TopicControllerTest.groovy

```diff
@@ -153,6 +153,26 @@ class TopicControllerTest extends Specification {
         extractJson(response, HttpStatus.INTERNAL_SERVER_ERROR) == [code: 500, message: "The topic with id '1' is archived"]
     }
 
+    def "#checkTopicExists not throws when topic exists"() {
+        when:
+        checkTopicExists(TOPIC_ID)
+
+        then:
+        1 * service.checkTopicExists(TOPIC_ID)
+        notThrown(TopicNotFoundException)
```

Comment:
`notThrown` looks like a redundant. The test fails anyway if something is thrown?
I would rename it to "#checkTopicExists returns success when topic exists"